plugins/woocommerce/templates/cart/cart.php

```diff
@@ -38,8 +38,9 @@
 
 			<?php
 			foreach ( WC()->cart->get_cart() as $cart_item_key => $cart_item ) {
-				$_product   = apply_filters( 'woocommerce_cart_item_product', $cart_item['data'], $cart_item, $cart_item_key );
-				$product_id = apply_filters( 'woocommerce_cart_item_product_id', $cart_item['product_id'], $cart_item, $cart_item_key );
+				$_product     = apply_filters( 'woocommerce_cart_item_product', $cart_item['data'], $cart_item, $cart_item_key );
+				$product_id   = apply_filters( 'woocommerce_cart_item_product_id', $cart_item['product_id'], $cart_item, $cart_item_key );
+				$product_name = apply_filters( 'woocommerce_cart_item_name', $_product->get_name(), $cart_item, $cart_item_key );
```

Comment:
There are some `$_product->get_name()` calls further down in the template that should be updated to `$product_name`.

Reply:
Correct, those are replaced too. Thanks https://github.com/woocommerce/woocommerce/pull/37830/commits/70cbbeebd422b7c592a49442dd31f83e976a9838